components/statuslight/index.css

```diff
@@ -11,243 +11,258 @@ governing permissions and limitations under the License.
-    border-radius: var(--mod-statuslight-corner-radius, var(--spectrum-statuslight-corner-radius));
-
-    --spectrum-statuslight-spacing-computed-top-to-dot: calc(var(--mod-statuslight-spacing-top-to-dot, var(--spectrum-statuslight-spacing-top-to-dot)) - var(--mod-statuslight-spacing-top-to-label, var(--spectrum-statuslight-spacing-top-to-label)));
-    margin-block-start: var(--spectrum-statuslight-spacing-computed-top-to-dot);
-    margin-inline-end: var(--mod-statuslight-spacing-dot-to-label, var(--spectrum-statuslight-spacing-dot-to-label));
-
-    /* support high contrast/forced-color-adjust mode for dot */
-    -ms-high-contrast-adjust: none;
-    forced-color-adjust: none;
-  }
-}
-
-.spectrum-StatusLight--neutral {
-  font-style: italic;
+	display: flex;
+	flex-direction: row;
+	align-items: flex-start;
+
+	min-block-size: var(--mod-statuslight-height, var(--spectrum-statuslight-height));
+
+	padding-block-start: var(--mod-statuslight-spacing-top-to-label, var(--spectrum-statuslight-spacing-top-to-label));
+	padding-block-end: var(--mod-statuslight-spacing-bottom-to-label, var(--spectrum-statuslight-spacing-bottom-to-label));
+	padding-inline: 0;
+	box-sizing: border-box;
+
+	font-size: var(--mod-statuslight-font-size, var(--spectrum-statuslight-font-size));
+	font-weight: var(--mod-statuslight-font-weight, var(--spectrum-statuslight-font-weight));
+	font-family: var(--mod-statuslight-font-family, var(--spectrum-statuslight-font-family));
+	font-style: var(--mod-statuslight-font-style, var(--spectrum-statuslight-font-style));
+
+	line-height: var(--mod-statuslight-line-height, var(--spectrum-statuslight-line-height));
+
+	color: var(--highcontrast-statuslight-content-color-default, var(--mod-statuslight-content-color-default, var(--spectrum-statuslight-content-color-default)));
+
+	&:lang(ja),
+	&:lang(zh),
+	&:lang(ko) {
+		line-height: var(--mod-statuslight-line-height-cjk, var(--spectrum-statuslight-line-height-cjk));
+	}
+
+	/* Dot */
+	&::before {
+		content: "";
+		flex-grow: 0;
+		flex-shrink: 0;
+		display: inline-block;
+		inline-size: var(--mod-statuslight-dot-size, var(--spectrum-statuslight-dot-size));
+		block-size: var(--mod-statuslight-dot-size, var(--spectrum-statuslight-dot-size));
+		border-radius: var(--mod-statuslight-corner-radius, var(--spectrum-statuslight-corner-radius));
+
+		--spectrum-statuslight-spacing-computed-top-to-dot: calc(var(--mod-statuslight-spacing-top-to-dot, var(--spectrum-statuslight-spacing-top-to-dot)) - var(--mod-statuslight-spacing-top-to-label, var(--spectrum-statuslight-spacing-top-to-label)));
+		margin-block-start: var(--spectrum-statuslight-spacing-computed-top-to-dot);
+		margin-inline-end: var(--mod-statuslight-spacing-dot-to-label, var(--spectrum-statuslight-spacing-dot-to-label));
+
+		/* support high contrast/forced-color-adjust mode for dot */
+		-ms-high-contrast-adjust: none;
+		forced-color-adjust: none;
```

Comment:
Just a note that I've moved the `forced-color-adjust` of the `::before` dot into the forced-colors media query [in this PR](https://github.com/adobe/spectrum-css/pull/2771/commits/ef65619aea6721aa3f69a45fd2481ca83af8a01c). And have removed the `-ms-high-contrast-adjust`.

Reply:
I tried to cherry pick the changes from that commit, so let me know if I missed anything!

https://github.com/adobe/spectrum-css/pull/2818/commits/b9b2b5019d6df761c55c13cb4330e4c204d82020